internal/provisioner/model/model.go

```diff
@@ -171,6 +171,14 @@ type ContourSpec struct {
 
 	// EnvoyWorkloadType is the way to deploy Envoy, either "DaemonSet" or "Deployment".
 	EnvoyWorkloadType WorkloadType
+
+	// KubernetesLogLevel Enable Kubernetes client debug logging with log level. If unset,
+	// defaults to 0.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=9
+	// +optional
```

Comment:
Don't need the kubebuilder markers here; if any other fields have them, they're just carried over from when this code was part of an API definition, but it is no longer so we don't need them.

Reply:
done